system/settings/settings.xml

```diff
@@ -4036,6 +4036,14 @@
           <level>4</level>
           <default>480</default>
         </setting>
+        <setting id="window.top" type="integer" label="0" help="39190">
+          <level>4</level>
+          <default>0</default>
+        </setting>
+        <setting id="window.left" type="integer" label="0" help="39191">
+          <level>4</level>
+          <default>0</default>
+        </setting>
```

Comment:
windows specific settings should be placed in https://github.com/xbmc/xbmc/blob/master/system/settings/windows.xml 
or hidden for non-windows platforms

Reply:
`window.width` and `window.height` settings already exists for all platforms here:

https://github.com/xbmc/xbmc/blob/5dda244747b86975de9ac4e104a4e981c535a509/system/settings/settings.xml#L4028-L4040

No seems much logical make now `window.top` and `window.left ` available only for Windows.

And these settings are already hidden:
https://github.com/xbmc/xbmc/blob/5dda244747b86975de9ac4e104a4e981c535a509/system/settings/settings.xml#L4029

I understand that it is used only to store the value but these settings are not visible in the GUI since the user does not need to manually modify the value (the modification occurs when the size or position of the window changes).